workflow-controller/controller-service/src/main/java/io/hyscale/controller/commands/deploy/HyscaleDeployServiceCommand.java

```diff
@@ -112,78 +128,93 @@
     private DeployComponentInvoker deployComponentInvoker;
     
     @Autowired
-    private ServiceSpecMapper serviceSpecMapper;
-
+    private HyscaleInputSpecProcessor hyscaleInputSpecProcessor;
+    
+    @Autowired
+    private PostValidatorsProvider postValidatorsProvider;
+    
+    @Autowired
+    private WorkflowContextBuilder workflowContextBuilder;
+    
     @Override
     public Integer call() throws Exception {
+        WorkflowLogger.header(ControllerActivity.PROCESSING_INPUT);
         if (!CommandUtil.isInputValid(this)) {
             return ToolConstants.INVALID_INPUT_ERROR_CODE;
         }
+        HyscaleCommandSpecBuilder commandSpecBuilder = new HyscaleCommandSpecBuilder();
```

Comment:
It is a model and not a builder , it is has to built a model then builder makes some sense here. You check this out once